### PR TITLE
Redesign vehicle selector

### PR DESCRIPTION
**Pull request by bbecquet**
## Description
Better look for vehicle selector, according to the latest designs.

## Screenshots
|Before|After|
|---|---|
|Capture d’écran 2020-10-06 à 11 01 03|Capture d’écran 2020-10-06 à 11 01 21|
|Capture d’écran de 2020-10-06 10-58-39|Capture d’écran de 2020-10-06 10-54-49|

### CODE REVIEW

File: src/scss/includes/vehicleSelector.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    &--active {
+      .vehicleSelector-buttonLabel {
+        color: $primary_text;
+      }
+
+      &:after{
+        content:'';
+        width: 100%;
+        height: 4px;
+        background: $focus-gradient;
```
Comment (G-Ray) [inline src/scss/includes/vehicleSelector.scss:72]: according to mockups, it's a blue gradient. Can we define it in `colors.scss`, like `focus-gradient` ?

Comment (bbecquet) [reply src/scss/includes/vehicleSelector.scss:72]: Yeah…

Comment (bbecquet) [reply src/scss/includes/vehicleSelector.scss]: Done :)

**Comment by G-Ray** [comment]
nice,

the only thing is we got the same hover issue as me on touch devices:
Peek 2020-10-06 16-23

On mobile devices, we can apply style only when element `active` for instance.

**Comment by bbecquet** [comment]
@G-Ray I don't define any `hover` on mobile. Your screenshot is on the desktop design.

**Comment by G-Ray** [comment]
> @G-Ray I don't define any `hover` on mobile. Your screenshot is on the desktop design.

my bad you r right...

Maybe we should review our hover effect for other touch devices (tablet devices) at one point.

**Comment by bbecquet** [comment]
@G-Ray The proper way is probably to use a feature detection media query.
https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Targeting_media_features

**Review by G-Ray — APPROVED** [review]

**Review by xem — APPROVED** [review]

**Pull request merged**